openpype/hosts/flame/plugins/publish/extract_subset_resources.py

```diff
@@ -159,7 +217,18 @@ def process(self, instance):
                 # add files to represetation but add
                 # imagesequence as list
                 if (
-                    "movie_file" in preset_path
+                    # first check if path in files is not mov extension
+                    next(
+                        # iter all paths in files
+                        # return only .mov positive test
+                        iter([
+                            f for f in files
+                            if ".mov" in os.path.splitext(f)[-1]
+                        ]),
+                        # if nothing return default
+                        None
+                    )
```

Comment:
ok, why this would be the reason for disapproval? Code is working and the `iter` and `next` are p3 and p2 compatible build in functions.. Can you elaborate little more about this? All code is annotated and clearly explained. In case you do not prefer next or `iter` you can explain any reason for not to use them.